app/controllers/user_tasks_controller.rb

```diff
@@ -3,4 +3,30 @@ def create
     UserTask.create!(task_id: params[:task_id], user_id: current_user.id)
     redirect_to dashboard_path
   end
+
+  def new
+    @user_task = UserTask.new
+  end
+
+  def edit
+    @user_task = UserTask.where( 'user_id = ? AND task_id = ?', @current_user.id, params[:id]).take
+    @task = Task.find(params[:id])
+  end
+
+  def update
+    @user_task = UserTask.where( 'user_id = ? AND task_id = ?', @current_user.id, params[:id]).take
+    @user_task.update(task_params)
+    @user_task.save!
+    redirect_to dashboard_path
+  end
+
+  def destroy
+    userTask = UserTask.where( 'user_id = ? AND task_id = ?', @current_user.id, params[:id]).take
```

Comment:
Za varijablu u destroyu? Ako je to samo problem, mergaj pa cu na masteru prepravit.